assets/js/modules/sign-in-with-google/index.js

```diff
@@ -62,5 +67,23 @@ export function registerModule( modules ) {
 				'google-site-kit'
 			),
 		],
+		checkRequirements: async ( registry ) => {
```

Comment:
I added HTTPS warning for the module as well, since I didn't see issue to include it, and it was quick to include it here as well